bin/powershell/check-windows-disk-writeable.ps1

```diff
@@ -51,7 +51,7 @@ $critDisks = ""
 
 $AllDisks = @()
 Foreach ($DT in $($DriveType -split(','))) {
-  Get-WMIObject Win32_LogicalDisk -Filter "DriveType = ${DT}" | Where-Object{ $_.DeviceID -notmatch "[$($Ignore.Replace(',',''))]:"} | %{ $AllDisks += $_ }
+  Get-CimInstance -ClassName Win32_LogicalDisk -Filter "DriveType = ${DT}" | ? { $_.DeviceID -notmatch "[$($Ignore.Replace(',',''))]:"} | %{ $AllDisks += $_ }
```

Comment:
Is there some value of `?` over `Where-Object`? I had to google it and they seem to be equivalent from what I (quickly) read. If thats the case I'd prefer to keep it more clear unless there is a performance penalty for it.